docs/security.html

```diff
@@ -909,21 +913,21 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
                                 as described <a href="#security_client_staticjaas">here</a>. Clients use the login section named
                                 <code>KafkaClient</code>. This option allows only one user for all client connections from a JVM.</p></li>
                         <li>Configure the following properties in producer.properties or consumer.properties:
-                            <pre><code class="language-text">security.protocol=SASL_SSL
+                            <pre><code class="language-text">security.protocol=SASL_PLAINTEXT
```

Comment:
Why are we changing this to `SASL_PLAINTEXT`?

Reply:
Reverted the change. Thanks.